tsuba: add write options struct

tylershunt opened the pull request:
For now let's you change the version of the output format. This is not
useful for reading and writing RDGs where we control both ends of the
operation, but it _is_ useful when we would like to use this interface
to write parquet files that should be usable by another consumer,
e.g., [BigQuery](https://issuetracker.google.com/issues/116699318).

Signed-off-by: Tyler Hunt <thunt@katanagraph.com>

ddn0 commented:
The configuration cruft the support this for real seems to outweigh the small annoyance of infrequent failed updates.

Commit:
tsuba: add write options struct

For now let's you change the version of the output format. This is not
useful for reading and writing RDGs where we control both ends of the
operation, but it _is_ useful when we would like to use this interface
to write parquet files that should be usable by another consumer,
e.g., [BigQuery](https://issuetracker.google.com/issues/116699318).

Signed-off-by: Tyler Hunt <thunt@katanagraph.com>
